static/app/components/events/interfaces/crashContent/exception/sourceMapDebug.tsx

```diff
@@ -54,6 +54,12 @@ function getErrorMessage(
     }
     return `${baseSourceMapDocsLink}troubleshooting_js/` + (section ? `#${section}` : '');
   }
+  function getMigrationGuide() {
+    if (docPlatform === 'react-native') {
+      return 'https://docs.sentry.io/platforms/react-native/migration/';
+    }
+    return 'https://github.com/getsentry/sentry-javascript/blob/develop/MIGRATION.md#upgrading-from-6x-to-7x';
```

Comment:
as this is static we will have to keep an eye out here otherwise it would lead the user to wrong information in the future